### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by WiserZhou** [microsoft/TRELLIS#329]
Hello,
        When I use your project, I encountered a problem: "Segmentation fault (core dumped)". I don't know why this will happen. Through my debug, I found that the code:`vmapping, indices, uvs = xatlas.parametrize(vertices, faces)` will throw this error. But I don't know why. And the program will be blocked due to this.

**Comment by Nepenthes-zlc** [comment]
I also have this problem, could you please tell me do you solve the problem?

**Comment by WiserZhou** [comment]
I still can't solve this.

**Comment by Nepenthes-zlc** [comment]
Do you use xformers? I have a friend, he uses flash-attn to run it without this error

**Comment by WiserZhou** [comment]
I installed them all

**Comment by xinnan-zhu** [comment]
Could you tell me which GPU you are using.

**Comment by WiserZhou** [comment]
H200 GPU

**Comment by xinnan-zhu** [comment]
> H200 GPU

I’m running on an H200 as well, and I’m experiencing the same problem.

**Comment by WiserZhou** [comment]
I haven't found any solution since then. Now I have to skip the mesh generation.

**Comment by wendashi** [comment]
🙋same problem on H200,,,  'Segmentation fault      (core dumped)'

**Comment by wendashi** [comment]
In my case, this works! (before, my `nvdiffrast` is 0.3.5)

```
pip install "https://huggingface.co/spaces/JeffreyXiang/TRELLIS/resolve/main/wheels/nvdiffrast-0.3.3-cp310-cp310-linux_x86_64.whl?download=true"
```

**Comment by WiserZhou** [comment]
It doesn't work for me. This is my package list, maybe someone can find the reason.
Package                     Version
--------------------------- ---------------
absl-py                     2.3.1
accelerate                  1.11.0
addict                      2.4.0
aiofiles                    23.2.1
aiohappyeyeballs            2.6.1
aiohttp                     3.13.2
aiosignal                   1.4.0
altair                      5.5.0
annotated-types             0.7.0
antlr4-python3-runtime      4.9.3
anyio                       4.11.0
argon2-cffi                 25.1.0
argon2-cffi-bindings        25.1.0
arrow                       1.3.0
asttokens                   3.0.0
async-lru                   2.0.5
async-timeout               5.0.1
attrs                       25.3.0
awscli                      1.42.68
babel                       2.17.0
bcrypt                      5.0.0
beautifulsoup4              4.13.5
bitsandbytes                0.48.2
bleach                      6.2.0
blessed                     1.23.0
blinker                     1.9.0
boto3                       1.40.68
botocore                    1.40.68
brainpp                     2.7.12.8
Brotli                      1.1.0
cachetools                  6.2.1
ccimport                    0.4.4
certifi                     2025.10.5
cffi                        2.0.0
charset-normalizer          3.4.3
click                       8.3.0
colorama                    0.4.6
coloredlogs                 15.0.1
comm                        0.2.3
ConfigArgParse              1.7.1
confluent-kafka             2.12.2
contourpy                   1.3.2
cryptography                46.0.3
cumm-cu120                  0.4.11
cumm-cu121                  0.7.11
cycler                      0.12.1
dash                        3.2.0
dataclasses-json            0.6.7
debugpy                     1.8.17
decorator                   5.2.1
defusedxml                  0.7.1
Deprecated                  1.3.1
diff_gaussian_rasterization 0.0.0
diffoctreerast              0.0.0
diffusers                   0.35.2
docutils                    0.19
dotmap                      1.3.30
durationpy                  0.10
easydict                    1.13
einops                      0.8.1
elasticsearch               7.9.1
entrypoints                 0.4
exceptiongroup              1.3.0
executing                   2.2.1
fastapi                     0.117.1
fastjsonschema              2.21.2
ffmpy                       0.6.1
filelock                    3.20.0
fire                        0.7.1
flash_attn                  2.8.3
Flask                       3.1.2
flatbuffers                 25.9.23
fonttools                   4.60.0
fqdn                        1.5.1
frozendict                  2.4.6
frozenlist                  1.8.0
fsspec                      2025.9.0
gitdb                       4.0.12
gitignore_parser            0.1.13
GitPython                   3.1.45
glcontext                   3.0.0
gmpy2                       2.2.1
google-auth                 2.43.0
gpustat                     1.1.1
gradio                      4.44.1
gradio_client               1.3.0
gradio_litmodel3d           0.0.1
grpcio                      1.75.1
h11                         0.16.0
h2                          4.3.0
hf-xet                      1.2.0
hpack                       4.1.0
httpcore                    1.0.9
httpx                       0.28.1
huggingface-hub             0.36.0
humanfriendly               10.0
hydra-core                  1.3.2
hyperframe                  6.1.0
idna                        3.11
igraph                      0.11.9
imageio                     2.37.0
imageio-ffmpeg              0.6.0
importlib_metadata          8.7.0
importlib_resources         6.5.2
install                     1.3.5
invoke                      2.2.1
iopath                      0.1.10
ipycanvas                   0.14.1
ipyevents                   2.0.4
ipykernel                   6.29.5
ipython                     8.37.0
ipython-genutils            0.2.0
ipywidgets                  8.1.7
isoduration                 20.11.0
itsdangerous                2.2.0
jedi                        0.19.2
Jinja2                      3.1.6
jmespath                    1.0.1
joblib                      1.5.2
json5                       0.12.1
jsonpointer                 3.0.0
jsonschema                  4.25.1
jsonschema-specifications   2025.9.1
jupyter_client              7.4.9
jupyter_core                5.8.1
jupyter-events              0.12.0
jupyter-lsp                 2.3.0
jupyter_server              2.17.0
jupyter_server_terminals    0.5.3
jupyterlab                  4.4.7
jupyterlab_pygments         0.3.0
jupyterlab_server           2.27.3
jupyterlab_widgets          3.0.15
kaolin                      0.18.0
kiwisolver                  1.4.9
kornia                      0.8.1
kornia_rs                   0.1.9
kubernetes                  34.1.0
lark                        1.3.0
lazy_loader                 0.4
lightning                   2.6.0
lightning-utilities         0.15.2
llvmlite                    0.45.0
lpips                       0.1.4
Markdown                    3.9
markdown-it-py              4.0.0
MarkupSafe                  2.1.5
marshmallow                 3.26.1
matplotlib                  3.10.6
matplotlib-inline           0.1.7
mdurl                       0.1.2
megfile                     2.2.10.post1
mistune                     3.1.4
moderngl                    5.12.0
mpmath                      1.3.0
msgpack                     1.1.1
multidict                   6.7.0
mypy_extensions             1.1.0
narwhals                    2.5.0
nbclassic                   1.3.3
nbclient                    0.10.2
nbconvert                   7.16.6
nbformat                    5.10.4
nest-asyncio                1.6.0
networkx                    3.4.2
ninja                       1.13.0
notebook                    6.5.7
notebook_shim               0.2.4
numba                       0.62.0
numpy                       2.2.6
nvdiffrast                  0.3.3
nvidia-cublas-cu12          12.1.3.1
nvidia-cuda-cupti-cu12      12.1.105
nvidia-cuda-nvrtc-cu12      12.1.105
nvidia-cuda-runtime-cu12    12.1.105
nvidia-cudnn-cu12           9.1.0.70
nvidia-cufft-cu12           11.0.2.54
nvidia-curand-cu12          10.3.2.106
nvidia-cusolver-cu12        11.4.5.107
nvidia-cusparse-cu12        12.1.0.106
nvidia-ml-py                13.580.82
nvidia-ml-py3               7.352.0
nvidia-nccl-cu12            2.20.5
nvidia-nvjitlink-cu12       12.9.86
nvidia-nvtx-cu12            12.1.105
oauthlib                    3.3.1
omegaconf                   2.3.0
onnxruntime                 1.23.0
open3d                      0.19.0
opencv-python-headless      4.12.0.88
orjson                      3.11.3
overrides                   7.7.0
packaging                   25.0
pandas                      2.3.2
pandocfilters               1.5.1
paramiko                    4.0.0
parso                       0.8.5
pccm                        0.4.16
pexpect                     4.9.0
pillow                      10.4.0
Pillow-SIMD                 9.5.0.post2
pip                         25.2
platformdirs                4.4.0
plotly                      6.3.0
plyfile                     1.1.2
pooch                       1.8.2
portalocker                 3.2.0
prometheus_client           0.23.1
prompt_toolkit              3.0.52
propcache                   0.4.1
protobuf                    6.32.1
psutil                      7.1.3
ptyprocess                  0.7.0
pure_eval                   0.2.3
py                          1.11.0
pyasn1                      0.6.1
pyasn1_modules              0.4.2
pybind11                    3.0.1
pycolmap                    3.13.0
pycparser                   2.23
pydantic                    2.11.9
pydantic_core               2.33.2
pydub                       0.25.1
pygltflib                   1.16.5
Pygments                    2.19.2
PyMatting                   1.1.14
pymeshfix                   0.17.1
PyNaCl                      1.6.0
pyOpenSSL                   25.3.0
pyparsing                   3.2.5
pyquaternion                0.9.9
PySocks                     1.7.1
python-dateutil             2.9.0.post0
python-json-logger          3.3.0
python-multipart            0.0.20
pytorch-lightning           2.6.0
pytorch3d                   0.7.8
pytz                        2025.2
pyvista                     0.46.3
PyYAML                      6.0.3
pyzmq                       27.1.0
ray                         2.49.2
referencing                 0.36.2
refile                      7.2.7.post3
regex                       2025.10.23
rembg                       2.0.67
requests                    2.32.5
requests-oauthlib           2.0.0
retry                       0.9.2
retrying                    1.4.2
rfc3339-validator           0.1.4
rfc3986-validator           0.1.1
rfc3987-syntax              1.1.0
rich                        14.1.0
rpds-py                     0.27.1
rsa                         4.7.2
ruff                        0.13.1
s3transfer                  0.14.0
safetensors                 0.6.2
scikit-image                0.25.2
scikit-learn                1.7.2
scipy                       1.15.3
scooby                      0.10.2
semantic-version            2.10.0
Send2Trash                  1.8.3
sentencepiece               0.2.1
sentry-sdk                  2.38.0
setuptools                  80.9.0
shellingham                 1.5.4
six                         1.17.0
smmap                       5.0.2
sniffio                     1.3.1
soupsieve                   2.8
spconv-cu120                2.3.6
spconv-cu121                2.3.8
stack-data                  0.6.3
starlette                   0.48.0
sympy                       1.14.0
tabulate                    0.9.0
tensorboard                 2.20.0
tensorboard-data-server     0.7.2
termcolor                   3.1.0
terminado                   0.18.1
texttable                   1.7.0
threadpoolctl               3.6.0
tifffile                    2025.5.10
timm                        1.0.21
tinycss2                    1.4.0
tokenizers                  0.22.1
tomli                       2.2.1
tomlkit                     0.12.0
torch                       2.4.0+cu121
torch_scatter               2.1.2+pt24cu121
torch_sparse                0.6.18
torchaudio                  2.4.0+cu121
torchmetrics                1.8.2
torchvision                 0.19.0+cu121
tornado                     6.5.2
tqdm                        4.67.1
traitlets                   5.14.3
transformers                4.56.2
trimesh                     4.8.2
triton                      3.0.0
typeguard                   4.4.4
typer                       0.19.2
typer-slim                  0.20.0
types-python-dateutil       2.9.0.20250822
typing_extensions           4.15.0
typing-inspect              0.9.0
typing-inspection           0.4.1
tzdata                      2025.2
uri-template                1.3.0
urllib3                     2.3.0
usd-core                    25.8
utils3d                     0.0.2
uvicorn                     0.37.0
vox2seq                     0.0.0
vtk                         9.5.2
wandb                       0.22.0
warp-lang                   1.9.0
wcwidth                     0.2.14
webcolors                   24.11.1
webencodings                0.5.1
websocket-client            1.8.0
websockets                  11.0.3
Werkzeug                    3.1.3
wheel                       0.45.1
widgetsnbextension          4.0.14
wrapt                       2.0.0
xatlas                      0.0.11
xformers                    0.0.27.post2
yarl                        1.22.0
zipp                        3.23.0
zstandard                   0.23.0

**Comment by uuunnnn** [comment]
you can try numpy==1.26.4
